nx-X11/programs/Xserver/hw/nxagent/Holder.c

```diff
@@ -23,6 +23,7 @@
 #include "scrnintstr.h"
 #include "Agent.h"
 #define GC XlibGC
+#define Pixmap XlibPixmap
```

Comment:
Why do we need this? It doesn't seem to actually fix any compiler warning, as far as I've seen. I didn't test on x86, though, only x86_64.